applications/nublado/values-idfdev.yaml

```diff
@@ -2,6 +2,17 @@ controller:
   googleServiceAccount: "nublado-controller@science-platform-dev-7696.iam.gserviceaccount.com"
   slackAlerts: true
   config:
+    dropdownMenu:
+      release:
+        cutoffVersion: "27.0.0"
+      weekly:
+        number: 78
+      daily:
+        number: 25
+      releaseCandidate:
+        cutoffVersion: "28.0.0"
+      experimental:
+        number: 5
```

Comment:
I think I'd rather see us use an age cutoff for experimental since I don't think the count is very meaningful.

Reply:
Sure thing.  I'll do one month.

Reply:
That's not a thing.  Three weeks.